### PR TITLE
CON-3392 Updated content for access denied

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/Provider/Provider.Web/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -19,7 +19,11 @@
     {
         const string defaultTitle = "Recruit an Apprentice";
         if (ViewBag.Title != null)
-            return $"{defaultTitle} | {ViewBag.Title}";
+        { 
+            return !(ViewBag.HideDefaultTitle ?? false)
```
Comment (cofaulco) [inline src/Provider/Provider.Web/Views/Shared/_Layout.cshtml:23]: might want to run this one by BA / content, if this is how all of Recruits page titles are handled

**Review by cofaulco — COMMENTED** [review]
1 comment

File: src/Provider/Provider.Web/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/Provider/Provider.Web/Views/Shared/_Layout.cshtml:23]: I did run it by Annukka - originally it was going to keep the default title as Paul said it was hard to remove - but they did want it to match - so I made this change

Comment (ChrisJWoodcock) [reply src/Provider/Provider.Web/Views/Shared/_Layout.cshtml:23]: Probably the whole of recruit should be changed - which would be easy enough as it's the odd one out

**Review by cofaulco — APPROVED** [review]

**Pull request merged**